src/pointer-analysis/value_set.h

```diff
@@ -163,7 +163,7 @@ class value_sett
   };
 
   /// Converts an `object_map_dt` entry `object_number -> offset` into an
-  /// `object_descriptor_offsett` with
+  /// `object_descriptor_exprt` with
   /// `.object() == object_numbering.at(object_number)` and
```

Comment:
If you are fixing up this comment, is there any milage in covering it to proper doxygen format (with `\param` notation) at the same time? Won't hold it against you if you don't want to though :-)

Reply:
I deliberately skipped `\param it: thing being converted; \return converted thing` cos I didn't think it added anything, but don't object if it helps the formatted docs? (I've never seen our actual Doxy output, so I'm flying blind a bit when writing this format)

Reply:
I've not actually seen them either, so equally blind :-) But I think it should help with the formatting and hyperlinking. But I really don't have a strong opinion here either.